pvfactors/tests/test_geometry/test_data/__init__.py

```diff
@@ -18,23 +18,24 @@
     dtype=int
 )
 
+
 vm_right_orderedpvarray = np.array(
     [[0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 9, 0, 9, 0, 9, 0, 1],
+     [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 7, 9, 0, 9, 0, 1],
+     [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 7, 0, 7, 9, 0, 1],
      [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 7, 0, 7, 9, 0, 1],
      [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 7, 0, 7, 0, 7, 1],
      [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 7, 0, 7, 0, 7, 1],
-     [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 7, 9, 0, 9, 0, 1],
-     [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 7, 0, 7, 9, 0, 1],
      [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 7, 0, 7, 0, 7, 1],
      [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 7, 0, 7, 9, 0, 1],
      [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 7, 0, 7, 0, 7, 1],
      [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 7, 0, 7, 0, 7, 1],
      [8, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 4],
      [0, 6, 6, 6, 6, 6, 6, 6, 6, 6, 0, 0, 10, 0, 0, 0, 5],
-     [8, 0, 0, 0, 8, 0, 0, 0, 0, 0, 0, 10, 0, 0, 0, 0, 4],
-     [0, 6, 6, 6, 0, 6, 6, 6, 6, 6, 0, 0, 0, 0, 10, 0, 5],
-     [8, 8, 0, 0, 8, 8, 0, 8, 0, 0, 0, 0, 0, 10, 0, 0, 4],
-     [0, 0, 6, 6, 0, 0, 6, 0, 6, 6, 0, 0, 0, 0, 0, 0, 5],
+     [8, 8, 0, 0, 0, 0, 0, 0, 0, 0, 0, 10, 0, 0, 0, 0, 4],
+     [0, 0, 6, 6, 6, 6, 6, 6, 6, 6, 0, 0, 0, 0, 10, 0, 5],
+     [8, 8, 8, 8, 0, 0, 0, 8, 0, 0, 0, 0, 0, 10, 0, 0, 4],
+     [0, 0, 0, 0, 6, 6, 6, 0, 6, 6, 0, 0, 0, 0, 0, 0, 5],
      [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]],
```

Comment:
the same surfaces are created, but their indexing differs now with the new implementation.
So the matrices are slightly shifted